microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md

```diff
@@ -30,10 +30,10 @@ An overview of the changes involved, as described in [Overview of Manifest V3 -
 
 To help you plan, consider the following schedule for Microsoft Partner Center and Microsoft Edge changes.
 
+We are currently in the process of updating our MV3 migration timelines.
 
-#### Revised timeline
+As of now, we will continue to follow Chromium’s MV3 timelines. While we fundamentally align with these timelines, we will continue to analyze the concerns raised by extension developers and explore the right path for Microsoft Edge Addons ecosystem as a whole. We will independently decide on MV3 migration timelines for Microsoft Edge Addons and share an update here once they are available.
```

Comment:
Revise this to as follows. 


We are aware that Chrome has revised the timelines for Manifest V2 sunset and as of now we will continue to follow Chromium's timelines. We continue to analyze the concerns raised by the extension developers and explore the optimal path for Microsoft Edge Add-ons ecosystem. We will independently decide on MV3 migration timelines for Microsoft Edge Addons and share an update here. Meanwhile, please refer to the [Chromium timelines ](https://developer.chrome.com/docs/extensions/mv3/mv2-sunset/) for planning your extension's migration.